app/models/concerns/hyrax/solr_document_behavior_decorator.rb

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+
+# OVERRIDE Hyrax 5.0.0rc2 to account for Valkyrie migration object that end in "Resource"
+
+module Hyrax
+  module SolrDocumentBehaviorDecorator
+    def hydra_model(classifier: nil)
```

Comment:
I don't like relying on the human_readable_type; that's subject to translation whims and I foresee a very quick change from `Generic Work resource` to `Generic Work`.

Consider that we have [`valkyrie_bsi` attribute on the Solr document](https://github.com/samvera/hyrax/blob/f4b8c28768314e690f9cff454184600e0ca49d29/app/models/concerns/hyrax/solr_document_behavior.rb#L80-L84); can we use that?